### ISSUE TITLE
[sort-ngmodule-metadata-arrays] false positive report issue

### ISSUE THREAD

**Issue by fchiumeo** [angular-eslint/angular-eslint#407]
**Description and reproduction of the issue**
new rule `sort-ngmodule-metadata-arrays` report false positive

```
error: NgModule metadata arrays should be sorted in ASC alphabetical order (@angular-eslint/sort-ngmodule-metadata-arrays) at src\app\sistema\sistema.module.ts:107:9:
  105 |         TipoServicioComponent,
  106 |         UsuarioComponent,
> 107 |         VariableComponent
      |         ^
  108 |     ],
  109 |     imports: [
  110 |         RouterModule.forChild(sistemaRoutes),
```

```typescript
@NgModule({
    declarations: [
        AuditoriaComponent,
        ClienteComponent,
        CompetidoraComponent,
        MercadoComponent,
        MotivoComponent,
        PeriodoFiscalComponent,
        ProveedorComponent,
        TareaLogComponent,
        TecnologiaComponent,
        TipoServicioComponent,
        UsuarioComponent,
        VariableComponent
    ],
    imports: [
        RouterModule.forChild(sistemaRoutes),
        SharedModule
    ]
})
```

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint       `        | `4.21.0` |
| `TypeScript`                       | `4.1.5` |
| `ESLint`                           | `7.24.0` |
| `node`                             | `15.14.0` |
